lib/ttnt/test_to_code_mapping.rb

```diff
@@ -9,10 +10,15 @@ module TTNT
   #   spectra: { filename => [line, numbers, executed], ... }
   #   mapping: { test_file => spectra }
   class TestToCodeMapping
+    STORAGE_SECTION = 'mapping'
+
     # @param repo [Rugged::Reposiotry] repository to save test-to-code mapping
     #   (only repo.workdir is used to determine where to save the mapping file)
-    def initialize(repo)
+    # @param sha [String] sha of commit from which mapping is read from.
```

Comment:
OMG :scream: Thank you for catching! Fixed: https://github.com/Genki-S/ttnt/commit/0e8d34d13ab08d6eb208b031cef67b8940a3ba48